components/bl-ace-editor-component/README.md

```diff
@@ -1,33 +1,54 @@
 # Ace Editor
 
-Ace Editor is a component of Backendless UI-Builder designer. The Ace Editor component is the high-performance code editor for the web. This component is created using the [react-ace](https://www.npmjs.com/package/react-ace) library.
+Ace is the embeddable code editor which supports the syntax highlighting for the following programming languages: Java, JavaScript, TypeScript, Python, JSON, PHP, C, C++, C#, Ruby, HTML, CSS.
+
+When this component is deployed it can highlight only one programming language that was set by developer in the Logic of the Backendless UI-Builder. Therefore, a user cannot switch to another programming language while using the editor. 
```

Comment:
> Therefore, a user cannot switch to another programming language while using the editor.

It's not completely true. We could add the Logic for change language dynamically.